packages/protocol/contracts/common/interfaces/IEpochManager.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.5.13 <0.9.0;
+
+interface IEpochManager {
```

Comment:
review methods here, i don't see all of them listed

Reply:
addressed by #11223 commit e94259d57e00e46593812fcbdb619f6baf9333fd